### PR TITLE
Add pull request instructions to readme

### PR DESCRIPTION
**Pull request by chelsabeth**
Add more detailed instructions to readme about how to make a pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,15 +30,15 @@ Cloning a project is similar to downloading a project. By cloning this project f
...
 4. Navigate to you're intended directory. To navigate to the desktop type cd Desktop/
 5. Type: ```git clone [URL GOES HERE]```
-6. Now that you have cloned the project down to your machine we need to navigate to inside that project. To do this type: ```cd Github-Practice/```
+6. Now that you have cloned the project down to your machine we need to navigate inside that project. To do this type: ```cd Github-Practice/```
 7. To confirm you have done these steps correctly when typing: ```git status``` it should tell you that you are on a specific branch.
 
 ### Create a new branch
 
-It's best practice to work off a develop branch, not master. 
+It's best practice to work off a develop branch, not master.
```
Comment (BCtopics) [inline README.md]: Could you change this from master to main?

**Review by BCtopics — COMMENTED** [review]
Incredible job!

File: README.md (hunk 1 above)
Comment (chelsabeth) [reply README.md]: Changed this! This PR should reflect those changes.

**Review by BCtopics — COMMENTED** [review]
perfect, thanks!

**Pull request merged**